src/js/playlist/loader.js

```diff
@@ -50,20 +53,22 @@ const PlaylistLoader = function() {
                         throw Error;
```

Comment:
If this is being passed in as a `sourceError`, we might want to give it a short message like "playlist is not an Array".